### DISCUSSION TITLE
Error while running lammps

### DISCUSSION THREAD

**Discussion by Sokseiha** [mir-group/pair_nequip#34] [Q&A] [answered]
Hello,

I would like to run lamps with NequIP pair style on a supercomputer that our group have been using and I was able to compile the code without (apparent) errors and generated the corresponding lmp executable. However, when I tried to run a calculation I got the error:
 'torch::jit::ErrorReport'
Does any of you have an idea of what it means and how to fix it?
Thanks a lot.

Best.

[slurm-44191189.txt](https://github.com/mir-group/pair_nequip/files/10406085/slurm-44191189.txt)

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @Sokseiha ,

It should be possible to get a much more complete error log. Can you try to get more information?

**Reply by Sokseiha** [reply]
Hi @Linux-cpp-lisp ,

I have tried a few different things and now I get a more 'understandable' error:
`ERROR: Pair style NEQUIP requires newton pair off (src/pair_nequip.cpp:108)`
But now if I set newton pair to off, I got cryptic errors:
`0: *** Error in `/users/smuy/lammps/build/lmp': malloc(): smallbin double linked list corrupted: 0x000000004d902d10 ***
1: *** Error in `/users/smuy/lammps/build/lmp': corrupted double-linked list: 0x0000000050f69a20 ***`
And nothing in the stderr.
I will keep investigating but if this rings anything to you, please let me know.

Best,
Seiha.

[lammps.log](https://github.com/mir-group/pair_nequip/files/10413851/lammps.log)

**Reply by Linux-cpp-lisp** [reply]
Hi Seiha,

OK, definitely progress towards something more informative, though not something I've ever seen.

Just confirming---  can this install of LAMMPS that you built with `pair_nequip` successfully run simulations without using `pair_nequip`? Like if you just run something with Lennard Jones as a test... want to be sure of whether the problem is indeed related to `pair_nequip`.  Like same system, same runscript, just a fake potential... isolate the cause

**Reply by Sokseiha** [reply]
Hi Alby,

After some more testings, I realized that the cryptic errors I mentioned in my previous message seems be caused by the fact that I trained the potential on a CPU architecture while I tried to use it to run MD on a GPU machine...
Anyway, the calculations are now running properly and the initial results seem promising :-)
Thank you very much for your prompt reply and wish you a good day.

Best,
Seiha.

**Reply by Linux-cpp-lisp** [reply]
Hi Seiha,

Huh, that should be supported, but I guess these things are often strangely environment dependent. Either way, I'm glad to hear that your calculations are going well!
